tests/integration/features/cleanup_int_test.go

```diff
@@ -109,3 +191,17 @@ func createdSecretHasOwnerReferenceToOwningFeature(namespace, secretName string)
 		}, tracker)
 	}
 }
+
+func namespaceExists(f *feature.Feature) bool {
+	namespace, err := fixtures.GetNamespace(f.Client, "conditional-ns")
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return false
+		}
+	}
```

Comment:
From the godoc: 

```golang
// IsNotFound returns true if the specified error was created by NewNotFound.
// It supports wrapped errors and returns false when the error is nil.
func IsNotFound(err error) bool 
```

so with that in mind, this should be sufficient:

```suggestion
	if errors.IsNotFound(err) {
		return false
	}
```